src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -211,9 +210,9 @@ module _ (S : Setoid c ℓ) where
 
   ∈-∃++ : ∀ {v xs} → v ∈ xs → ∃₂ λ ys zs → ∃ λ w →
           v ≈ w × xs ≋ ys ++ [ w ] ++ zs
-  ∈-∃++ (here px)                  = [] , _ , _ , px , ≋-refl
-  ∈-∃++ (there {d} v∈xs) with ∈-∃++ v∈xs
-  ... | hs , _ , _ , v≈v′ , eq = d ∷ hs , _ , _ , v≈v′ , refl ∷ eq
+  ∈-∃++ (here px)        = [] , _ , _ , px , ≋-refl
+  ∈-∃++ (there {d} v∈xs) = let hs , _ , _ , v≈v′ , eq = ∈-∃++ v∈xs
+                           in d ∷ hs , _ , _ , v≈v′ , refl ∷ eq
```

Comment:
This indentation style is inconsistent with the rest of the changes.

Reply:
Hopefully commit 66a6207a14332f7e3ff15d9e15dc196159f5ace8 fixes this appropriately.